framework/include/cppmicroservices/Bundle.h

```diff
@@ -430,7 +430,7 @@ class US_Framework_EXPORT Bundle
    *
    * @see Constants#BUNDLE_LOCALIZATION
    */
-  AnyMap GetHeaders() const;
+  const AnyMap& GetHeaders() const;
```

Comment:
a note describing the lifetime guarantee for the returned ```AnyMap&``` would be helpful to users.